prow/integ-suite-ocp.sh

```diff
@@ -0,0 +1,124 @@
+#!/bin/bash
+
+# Copyright 2019 Istio Authors
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#    http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+# This script is used to run the integration tests on OpenShift.
+# Usage: ./integ-suite-ocp.sh TEST_SUITE SKIP_TESTS, example: /prow/integ-suite-ocp.sh telemetry "TestClientTracing|TestServerTracing"
```

Comment:
can we adjust the parameters so it behaves exactly as integ-suite-kind?

Reply:
Yes, I think we can have it. I'll check and tell you something

Reply:
Added a TODO here to address this in another PR later when we have the Skip flag in the Make test file